ci/jenkins/Jenkins_steps.groovy

```diff
@@ -161,55 +161,55 @@ def compile_unix_mkl_cpu(lib_name) {
     }]
 }
 
-def compile_unix_mkldnn_cpu(lib_name) {
+def compile_unix_onednn_cpu(lib_name) {
     return ['CPU: MKLDNN': {
       node(NODE_LINUX_CPU) {
         ws('workspace/build-mkldnn-cpu') {
```

Comment:
Please consider renaming also workspace names

Reply:
It is already listed is proposed changes.